hack/testing/logging.sh

```diff
@@ -491,6 +491,7 @@ os::cmd::expect_success "oc patch $ops_dc \
    -p '{\"spec\":{\"template\":{\"spec\":{\"containers\":[{\"name\":\"elasticsearch\",\"securityContext\":{\"privileged\": true}}]}}}}'"
 
 os::cmd::try_until_text "oc deploy $ops_dc --latest" "Started" "$(( 3 * TIME_MIN ))"
+oc logs -f $ops_dc
```

Comment:
In general I'd recommend try_until over just oc logs - it'll be safer.

On Tue, Jul 5, 2016 at 10:34 AM, Richard Megginson <notifications@github.com
> wrote:

> In hack/testing/logging.sh
> <https://github.com/openshift/origin-aggregated-logging/pull/192#discussion_r69573398>
> :
>
> > @@ -491,6 +491,7 @@ os::cmd::expect_success "oc patch $ops_dc \
> >     -p '{\"spec\":{\"template\":{\"spec\":{\"containers\":[{\"name\":\"elasticsearch\",\"securityContext\":{\"privileged\": true}}]}}}}'"
> >
> >  os::cmd::try_until_text "oc deploy $ops_dc --latest" "Started" "$(( 3 * TIME_MIN ))"
> > +oc logs -f $ops_dc
>
> I don't know how it works, but oc logs -f $something will exit as soon as
> the pod or task is completed (success or failure). I guess we could ask
> @smarterclayton <https://github.com/smarterclayton> if using logs -f is
> an effective way to wait until certain pods/tasks are complete.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/192/files/865643dbb69290e4df1339fc7ae905fb2d0d269d#r69573398>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p6xZ5BZgtqq65DJrNYyutEI-9odEks5qSmtpgaJpZM4JDiu->
> .
>